Gemfile

```diff
@@ -46,7 +46,7 @@ gem 'kaminari'
 gem 'kramdown', '~> 2.1'
 gem 'local_time', '~> 2.1'
 gem 'mercury-rails', git: 'https://github.com/AgileVentures/mercury.git'
-gem 'nokogiri', '1.12.0'
+gem 'nokogiri', '1.14.2'
```

Comment:
I think we want: `gem 'nokogiri', '~> 1.14.2'`

Also, it seems to get 'bundle' to work on docker we need: `gem 'nokogiri', '~> 1.14.2', force_ruby_platform: true` (but doesn't work on my mac).

We can probably live with this for now by creating an Issue, and putting these instructions in the docker README.md.